src/Controllers/CopyPatrol.php

```diff
@@ -156,7 +160,7 @@ public function getReportLink( $ithenticateId ) {
 	 */
 	public function formatTimestamp( $datetime ) {
 		$datetime = strtotime( $datetime );
-		return date( 'd-m-y h:m:s', $datetime );
+		return date( 'Y-m-d h:m', $datetime );
```

Comment:
Assuming this was fine, going off of the [wireframe](https://phab.wmfusercontent.org/file/data/7i6yksmlqna7ri4mywe5/PHID-FILE-abfr7ggyussiagpan7a3/copy_patrol_3_-_review_selected.jpg)